### ISSUE TITLE
How do I change jest unit test timeout ?

### ISSUE THREAD

**Issue by sylvainpolletvillard** [codesandbox/codesandbox-client#1235]
# ❔ Question

I need to change the default timeout (5000ms) of a Jest unit test. The error states :
> Exceeded timeout of 5000ms for a test. Use jest.setTimeout(newTimeout) to increase the timeout value, if this is a long-running test.

but then I got `jest.setTimeout is not a function` error.

How do I override the timeout for the unit tests ? (globally or per test, I don't care)

### Your Environment
| Software         | Name/Version|
| ---------------- | ---------- |
| Сodesandbox  | latest
| Browser | Chrome 70
| Operating System | Windows 7

**Comment by Komar0ff** [comment]
Hi, **@sylvainpolletvillard**, what template for project do you use?

**Comment by sylvainpolletvillard** [comment]
Hello @Komar0ff ,

I use the `parcel` template

**Comment by lbogdan** [comment]
Hey @sylvainpolletvillard , can you please provide us with a sandbox in which this happens?

**Comment by sylvainpolletvillard** [comment]
I made a new basic reproduction test here : https://codesandbox.io/s/1orx48309l

Basically my question is for any test that has a legit reason to take over 5000 ms. In my usecase, I do a JavaScript training exercise where trainees have to handle different async operations, that take more than 5 seconds in total. I solved it by reducing the delay of async operations, but my question remains unanswered.
